selfdrive/car/gm/carcontroller.py

```diff
@@ -115,6 +120,9 @@ def update(self, sendcan, enabled, CS, frame, actuators, \
       if self.car_fingerprint == CAR.VOLT:
         can_sends.append(gmcan.create_steering_control(self.packer_pt,
           canbus.powertrain, apply_steer, idx, lkas_enabled))
+      if self.car_fingerprint == CAR.ACADIA_DENALI:
```

Comment:
same as above

Reply:
i duplicated alot of the settings as a starting point. im really not sure what will be different from the volt but i know some settings will need to be different from the volt.

 i am having an issue with steering temp unavailable message after different amounts of time.  im wondering if it has to do with the volt being electric? maybe something to do with regen breaking throwing errors? im still very new to openpilot so any help would be greatly appreciated

https://community.comma.ai/cabana/?route=7cc2a8365b4dd8a9%7C2018-08-28--19-07-07&max=6&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2F7cc2a8365b4dd8a9%2F2d522ddc11f1292d5dd307d63aa0321b_2018-08-28--19-07-07